introduction/what-is.md

```diff
@@ -0,0 +1,48 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+Let's say that team A uses software produced by team B.
+Team A submits a feature request to team B, but team B isn't able to implement that feature in time for team A.
+In an InnerSource setting, if team A can't get this feature request then it submits a pull request instead.
+That is to say team A implements the feature directly in team B's software and submits a pull request with the code changes.
+Team B partners to review and accept the submitted code.
+
+In this example, we call team A the **Guest** team and team B the **Host** team.
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
+In that situation, most people want to be a good host.
+They ensure that things are kept neat and tidy in anticipation of their guests' arrival.
+Visitors are greeted at the door and invited in.
+They can use the features and utilities that are in the home's public areas.
+There may be a few house rules that guests are asked to follow.
+Similarly, most guests want to show respect for the home and their host.
+They're careful with the items in the house and follow the rules for the duration of their stay.
+They may hope for or expect a return invitation as long as they've been courteous and polite.
+These concepts around a home visit are a metaphor for the attitude and behaviors that teams should bring as one hosts another making a guest contribution to the codebase.
+
+Let's take a closer look at how the mechanics of the inner source process can work.
+To aid in this explanation, we'll name a few key individuals on the guest and host teams.
+First, the **Product Owner** determines what functionality the host team is willing to accept as a contribution.
+The **Contributor** is the individual on the guest team that submits the code contribution for review by the host team.
+The **Trusted Committer** represents the host team in providing any timely support and mentorship that the contributor needs to successfully submit the pull request.
+
+With those definitions, here is the basic outline of an inner source contribution.
+
+* Guest team or contributor requests a feature from the host team.
+* Product owner ensures that user stories representing the feature request are created.
```

Comment:
With the more grass-roots style projects: I would argue that guests might file tickets themselves, get a bunch of feedback around filing it and start bringing up the matching PR's. 
The sentence could be understood as in "ensures that it gets done either by host or guest" or "hosts create them". Could we nudge the sentence a bit more into the "can be created by either side" direction?
```suggestion
* Product owner ensures that user stories representing the feature request are created, either by members of the guest team or host team.
```